common/src/main/java/net/xolt/freecam/config/ModConfig.java

```
@@ -95,6 +101,17 @@ public static class UtilityConfig {
         public InteractionMode interactionMode = InteractionMode.CAMERA;
     }
 
+    @ConfigEntry.Gui.Tooltip
+    @ConfigEntry.Gui.CollapsibleObject
+    public ServerConfig servers = new ServerConfig();
+    public static class ServerConfig {
+        @ConfigEntry.Gui.Tooltip(count = 2)
+        @ConfigEntry.Gui.EnumHandler(option = EnumDisplayOption.BUTTON)
+        public ServerRestriction mode = ServerRestriction.NONE;
+        public List<String> whitelist = Collections.emptyList();
+        public List<String> blacklist = Collections.emptyList();
```

These should be initialized with a mutable list instead of `Collections.emptyList()`.

Turns out autoconfig tries to mutate the existing list on load, and fails silently.